docs/src/userguide.md

```diff
@@ -28,22 +28,47 @@ To construct a sparse approximation to the exact posterior, we first need to sel
+The inducing inputs `z` imply some latent function values `u = f(z)`, sometimes called pseudo-points. The [`SparseVariationalApproximation`](@ref) specifies a distribution `q(u)` over the pseudo-points. In the case of GP regression, the optimal form for `q(u)` is a multivariate Gaussian, which is the only form of `q` currently supported by this package.
 ```julia
 using Distributions, LinearAlgebra
 q = MvNormal(zeros(length(z)), I)
 ```
 Finally, we pass our `q` along with the inputs `f(z)` to obtain an approximate posterior GP:
 ```julia
 fz = f(z, 1e-6)  # 'observe' the process at z with some jitter for numerical stability 
-approx = SVGP(fz, q)  # Instantiate everything needed for the svgp approximation
+approx = SparseVariationalApproximation(fz, q)  # Instantiate everything needed for the approximation
 
-svgp_posterior = posterior(approx)  # Create the approximate posterior
+sva_posterior = posterior(approx)  # Create the approximate posterior
 ```
 
 ## The Evidence Lower Bound (ELBO)
 The approximate posterior constructed above will be a very poor approximation, since `q` was simply chosen to have zero mean and covariance `I`. A measure of the quality of the approximation is given by the ELBO. Optimising this term with respect to the parameters of `q` and the inducing input locations `z` will improve the approximation.
 ```julia
-elbo(SVGP(fz, q), fx, y)
+elbo(SparseVariationalApproximation(fz, q), fx, y)
 ```
 A detailed example of how to carry out such optimisation is given in [Regression: Sparse Variational Gaussian Process for Stochastic Optimisation with Flux.jl](@ref). For an example of non-conjugate inference, see [Classification: Sparse Variational Approximation for Non-Conjugate Likelihoods with Optim's L-BFGS](@ref).
+
+# Available Parametrizations
+
+Two parametrizations of `q(u)` are presently available: [`Centered`](@ref) and [`NonCentered`](@ref).
+The `Centered` parametrization expresses `q(u)` directly in terms of its mean and covariance.
+The `NonCentered` parametrization instead parametrizes the mean and covariance of
+`ε := cholesky(cov(u)).U' \ (u - mean(u))`.
+These parametrizations are also known respectively as "Unwhitened" and "Whitened".
+
+The choice of parametrization can have a substantial impact on the time it takes for ELBO
+optimisation to converge, and which parametrization is better in a particular situation is
+not generally obvious.
+That being said, the `NonCentered` parametrization often converges in fewer iterations, so it is the default --
+it is what is used in all of the examples above.
+
+If you require a particular parametrization, simply use the 3-argument version of the
+approximation constructor:
+```julia
+SparseVariationalApproximation(Centered(), fz, q)
+SparseVariationalApproximation(NonCentered(), fz, q)
+```
+
+For a discussion around these two parametrizations, see e.g. [^Gorinova]
```

Comment:
One aspect that Gorinova et al are perhaps not discussing specifically is that, when considering GPs, there's a lot of correlation imposed by the kernel on `u = f(Z)`, section 3.4 in https://www.stat.berkeley.edu/~paciorek/diss/paciorek-thesis.pdf discusses that more concretely for the GP case, though I'm not sure if it's the _best_ reference on this topic, maybe worth including as well?